internal/commands/bake.go

```diff
@@ -176,7 +176,7 @@ type Bake struct {
 		Sha256                   bool     `            long:"sha256"                                                description:"calculates a SHA256 checksum of the output file"`
 		StubReleases             bool     `short:"sr"  long:"stub-releases"                                         description:"skips importing release tarballs into the tile"`
 		Version                  string   `short:"v"   long:"version"                                               description:"version of the tile"`
-		SkipFetchReleases        []string `short:"sfr" long:"skip-fetch-directories"        description:"skips the automatic release fetch the specified release directories"`
+		SkipFetchReleases        bool     `short:"sfr" long:"skip-fetch-directories"                                description:"skips the automatic release fetch for all release directories"`
```

Comment:
Would you please introduce a new flag "--skip-fetch" and "--skip-fetch-directories" as an alias to it. https://github.com/pivotal-cf/kiln/tree/feature-export-bosh-release-download-methods

Reply:
Done. I'd suggest testing this locally before merging